code/sorting/View/build.gradle

```diff
@@ -13,4 +13,5 @@ javafx {
 
 dependencies {
 	compile project(':sorting-ViewModel')
+    compile project(':sorting-Infrastructure')
```

Comment:
check string indentation

Reply:
It's looks like github code presentation issue, in my local files this lines on one level.

Reply:
validate the symbols which are exactly used. \t or space.

Reply:
Fixed only on git side. idk how it works :D